api/src/main/java/org/openmrs/Form.java

```diff
@@ -9,32 +9,57 @@
  */
 package org.openmrs;
 
+import javax.persistence.AttributeOverride;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
 import java.util.ArrayList;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
-
 /**
  * Form
  *
  * @version 1.0
  */
+@Entity
+@Table(name = "form")
+@AttributeOverride(name = "retireReason", column = @Column(name = "retired_reason"))
 public class Form extends BaseChangeableOpenmrsMetadata {
```

Comment:
@alinmihaila99  looking good to me,@Ruhanga how does this look to you.

Reply:
Thanks @alinmihaila99, @HerbertYiga. This is good.